### PR TITLE
corrected cp command

### PR DESCRIPTION
**Pull request by microSharjeel**

### CODE REVIEW

File: hardware/asic/skywater/Makefile [hunk 1]
```diff
@@ -40,7 +40,7 @@ $(DEFINE): sw boot.hex
 	cp boot.hex temp_src
 
 synth:  clean $(VSRC) $(VHDR) temp_src temp_inc $(DEFINE) 
-	cp /opt/OpenLane . 
+	cp -r /opt/OpenLane .
```
Comment (jjts) [inline hardware/asic/skywater/Makefile:43]: cannot have have hardcoded paths as users may use different paths

pls call it OPENLANE_SRC_DIR

**Comment by jjts** [comment]
do you have more hardcoded stuff ?

**Comment by jjts** [comment]
also, change the readme so users set the right env variables

File: hardware/asic/skywater/Makefile (hunk 1 above)
Comment (microSharjeel) [reply hardware/asic/skywater/Makefile:43]: OPENLANE_SRC_DIR 
should it be inside Makefile only or it should be in bash?

**Comment by microSharjeel** [comment]
no more hardcoded stuff i believe

File: hardware/asic/skywater/Makefile (hunk 1 above)
Comment (jjts) [reply hardware/asic/skywater/Makefile:43]: how can it be in makefile without being hardcoded?

Comment (jjts) [reply hardware/asic/skywater/Makefile:43]: the user must define it -- write that in the readme

**Comment by microSharjeel** [comment]
In README it is already generalized like see following statements
export OPENLANE_HOME=path/to/OpenLane/root
export PDK_ROOT=$OPENLANE_HOME/pdks

File: hardware/asic/skywater/Makefile (hunk 1 above)
Comment (microSharjeel) [reply hardware/asic/skywater/Makefile:43]: export OPENLANE_SRC_DIR=<path to openlane src directory>
I have added to README
and this will be in bash now

**Comment by microSharjeel** [comment]
check now changes in README and Makefile now appear in PR

**Pull request closed without merging**